src/megaapi_impl.cpp

```diff
@@ -16178,6 +16178,7 @@ void MegaApiImpl::processTransferFailed(Transfer *tr, MegaTransferPrivate *trans
     transfer->setLastError(megaError);
     transfer->setPriority(tr->priority);
     transfer->setState(MegaTransfer::STATE_RETRYING);
+    transfer->setStreamingTransfer(tr->mForeignTarget);
```

Comment:
'streaming transfer' refers to whether the transfer was started with a call to `MegaApi::startStreaming`.   I think we should make a new appropriately named getter rather than overloading this setter/getter with a meaning that doesn't match the name?   Also overquota errors could occur for streaming or non-streaming in which case we won't know which of those reasons this flag was set?  thanks

Reply:
Hi, Matt, we try to reuse class members if it's possible in order to not make this kind of classes bigger. This error only occurs for not streaming transfers so we thought we could use this member to indicate the error.

Reply:
Yes, we try to reuse existing flags / getter inside `MegaRequest` and alike, since the class is multi-purpose and adding a new getter or a new member for every different usage would result on a even longer list.

In this case, I suggested the _streaming-transfer_ since it's a flag and it assumed it was not in use for uploads (only downloads can be streaming. However, a quick search of `MegaTransfer::isStreamingTransfer()`... and boom, it's used at `MegaApiImpl::getStreamingTransfers()`, so we would start returning uploads in the set of streaming transfers --> bug!

Please, @jgandres , proceed with the addition of a new flag + setter/getter and adjust the code accordingly, please (and the documentation). Thanks

Reply:
Sure, I'll adjust now